weblate/utils/search.py

```diff
@@ -507,6 +507,8 @@ def has_field(self, text: str, context: dict) -> Q:  # noqa: C901
                     "|".join(re_escape(term) for term in terms)
                 )
             )
+        if text == "pending":
+            return Q(pending_changes__isnull=False)
```

Comment:
This seems extra, it changes the `has` lookup which does not have `pending` option.

```suggestion
```